.github/workflows/qiita-plugin-ci.yml

```diff
@@ -176,7 +176,7 @@ jobs:
     - name: flake8
       uses: actions/setup-python@v2
       with:
-        python-version: 3.6
+        python-version: 3.9
```

Comment:
@antgonza is changing to 3.9 okay? I thought plugins required 3.6

Reply:
That's _not_ for plugins, that's for flake8 and it needs 3.9.

Reply:
Ok.